src/js/actions/documents.js

```diff
@@ -826,28 +827,40 @@ define(function (require, exports) {
      */
     var handlePlaceEvent = function (event) {
         var applicationStore = this.flux.store("application"),
+            policyStore = this.flux.store("policy"),
             document = applicationStore.getCurrentDocument(),
-            replace = false;
-
-        if (!document) {
-            var error = new Error("Place event received without a current document");
-            return Promise.reject(error);
+            replace = false,
+            promise = Promise.bind(this);
+            
+        if (policyStore.areAllSuspended()) {
+            // If the policies are all suspended for drag-n-drop graphic asset, we restore them after the 
+            // asset is placed.
+            promise = promise.then(function () {
```

Comment:
I think more commonly in our code we would see: `promise = this.transfer(...`   and then a few lines below: `} else { promise = Promise.resolve():`   

... just a suggestion.